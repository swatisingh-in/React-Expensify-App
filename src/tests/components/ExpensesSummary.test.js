import React from 'react';
import { shallow } from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('Should render ExpensesSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={3} expensesTotal={231321331}/>);
  expect(wrapper).toMatchSnapshot();
});

test('Should render ExpensesSummary with single expense', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={10}/>);
  expect(wrapper).toMatchSnapshot();
});
