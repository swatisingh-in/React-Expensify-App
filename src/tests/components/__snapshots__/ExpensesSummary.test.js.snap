// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totalling 
    $2,313,213.31
  </h1>
</div>
`;

exports[`Should render ExpensesSummary with single expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $0.10
  </h1>
</div>
`;
